packages/flutter_tools/lib/src/dart/package_map.dart

```diff
@@ -17,6 +17,16 @@ Map<String, Uri> _parse(String packagesPath) {
 class PackageMap {
   PackageMap(this.packagesPath);
 
+  static PackageMap createGlobalInstance() => new PackageMap(globalPackagesPath);
```

Comment:
Does this need to create a new instance on every call?  Or can this be a lazily created singleton?